src/main/java/Dao/Mail.java

```diff
@@ -0,0 +1,62 @@
+package Dao;
+
+import javax.mail.*;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import java.util.Properties;
+
+public class Mail {
+    public static void main(String[] args) throws MessagingException {
+        System.out.println("prepararing send to message");
+        String message="Hello,thanks for Register ";
+        String subject="Well com Profile ";
+        String to="priyesh.jn786@gmail.com";
+        String from="priyesh.bhawsar@hotwaxsystems.com";
+
+
+        sentEmail(message,subject,to,from);
+    }
+        //this is responsible  to send email
+        private static void sentEmail(String message, String subject, String to, String from) throws MessagingException {
+        //variable for gmail
+        String host = "smtp.gmail.com";
+        //get the syatem.properties
+        Properties properties = System.getProperties();
+        System.out.println("Properties" + properties);
+        //setting importatent information  to properties object
+        //host set
+        properties.put("mail.smtp.host", host); //host key value pair
+        properties.put("mail.smtp.port", "465"); //port key value pair
+        properties.put("mail.smtp.ssl.enable", "true"); //ssl enable key value pair
+        properties.put("mail.smtp.auth", "true"); // auth enable key value pair
+
+        //step1: to get the session object..
+        Session session = Session.getInstance(properties, new Authenticator() {
+            @Override
+            protected PasswordAuthentication getPasswordAuthentication() {
+                return new PasswordAuthentication("priyesh.bhawsar@hotwaxsystems.com", "Shrinatraj786@");
```

Comment:
This is a bad practice. Please don't use your credentials in the Java files. A few days ago I shared an option to do so:
https://www.baeldung.com/java-properties
https://www.geeksforgeeks.org/reading-and-writing-properties-file-in-java/